applications/tari_dan_wallet_daemon/src/jrpc_server.rs

```diff
@@ -46,6 +46,17 @@ pub async fn listen(
 
 async fn handler(Extension(context): Extension<Arc<HandlerContext>>, value: JsonRpcExtractor) -> JrpcResult {
     info!(target: LOG_TARGET, "🌐 JSON-RPC request: {}", value.method);
+    if value.method == "rpc.discover" {
```

Comment:
We use the `.` now so you could add this to the handlers below - I dont mind either way